bundles/org.openhab.io.homekit/README.md

```diff
@@ -828,8 +877,36 @@ or using UI
+|                      |                             | ConfiguredName               | String                        | Name of the input source configured in the iOS Home app. User can rename the source in iOS Home app and this characteristic can be used to reflect change in openHAB and sync name changes from openHAB to Home app. |
+|                      |                             | Configured                   | Switch, Contact, Dimmer       | If the source is configured on the device. Non-configured inputs will not show up in the Home app. - ON/OPEN = show, OFF/CLOSED = hide. Default is ON. Can also be configured via metadata, e.g. [Configured=true] |
+|                      |                             | InputSourceType              | String                        | Type of the input source. possible values (OTHER, HOME_SCREEN, TUNER, HDMI, COMPOSITE_VIDEO, S_VIDEO, COMPONENT_VIDEO, DVI, AIRPLAY, USB, APPLICATION). Custom mapping can be defined at item level. Can also be configured via metadata, e.g. [InputSourceType="OTHER"]. |
+|                      |                             | CurrentVisibility            | Switch, Contact, Dimmer       | If the source has been hidden by the user - ON/OPEN = visible, OFF/CLOSED = hidden. Default is ON. Can also be configured via metadata, e.g. [CurrentVisibility=false] |
+|                      |                             | Identifier                   | Number                        | The identifier of the source, to be used with the ActiveIdentifier characteristic. Can also be configured via metadata, e.g. [Identifier=1] |
+|                      |                             | InputDeviceType              | String                        | Type of the input device. possible values (OTHER, TV, RECORDING, TUNER, PLAYBACK, AUDIO_SYSTEM). Custom mapping can be defined at item level. Can also be configured via metadata, e.g. [InputDeviceType="OTHER"]. |
+|                      |                             | TargetVisibilityState        | Switch                        | The desired visibility state of the input source. ON = shown, OFF = hidden. |
+| TelevisionSpeaker    |                             |                              |                               | An accessory that can be added to a Television in order to control the speaker associated with it. |
+|                      | Mute                        |                              | Switch                        | If the television is muted. ON = muted, OFF = not muted. |
+|                      |                             | Active                       | Switch                        | ?? |
+|                      |                             | Volume                       | Dimmer, Number                | Current volume. min/max/step can configured at item level, e.g. minValue=10.5, maxValue=50, step=2] |
```

Comment:
```suggestion
|                      |                             | Volume                       | Dimmer, Number                | Current volume. Min/max/step can configured at item level, e.g. minValue=10.5, maxValue=50, step=2] |
```

Reply:
you mind if I leave this as-is for now? it matches the rest of the table. a subsequent pending PR already re-organizes configuration values in this table, and removes this wording

Reply:
OK.